src/random.cr

```diff
@@ -56,6 +56,14 @@ module Random
   # The integers must be uniformly distributed between 0 and the maximal value for the chosen type.
   abstract def next_u : UInt
 
+  # Generates a slice filled with *n* random bytes.
+  def random_bytes(n : Int) : Bytes
```

Comment:
`IO#read` takes a slice of a fixed size and fills it. It's designed so that you can reuse the slice efficiently.

This takes a size and returns a slice of that size, which means that the allocation isn't controlled by the caller. I know why you do this for performance when filling it with integers, but I dislike the inconsistency.